services/QuillLMS/engines/evidence/app/views/evidence/research/gen_ai/application/_confusion_matrix.html.erb

```diff
@@ -12,14 +12,27 @@
       <td class='light-green' style="text-align: center;"><%= matrix[0][0] %></td>
       <td class='dark-red' style="text-align: center;"><%= matrix[0][1] %></td>
       <td class='white' style="text-align: center;"><%= matrix[0][0] + matrix[0][1] %></td>
-      <td class='white' style="text-align: center;"><%= (100.0 * matrix[0][0] / (matrix[0][0] + matrix[0][1])).round.to_s + '%' %></td>
+      <td class='white' style="text-align: center;">
+        <% if matrix[0][0] + matrix[0][1] == 0 %>
```

Comment:
Small note: You might consider making this a method on the model or a helper method. As a new set of eyes, it's somewhat opaque what this logic is calculating, and might be hard for someone else to debug. A name might help.